LeetCode/vAnil/String/StringPermutation/StringPermutation.cpp

```diff
@@ -0,0 +1,55 @@
+#include "StringPermutation.h"
+#include <unordered_map>
+#include <array>
+
+
+
+void getStringPermutations(std::vector<char> const &charArray, std::vector<size_t> &countArray, std::vector<std::string>& stringVec, size_t const strLenght, size_t level, std::string &result)
+{
+	if (level >= strLenght)
```

Comment:
try...catch missing.

Reply:
This is good to point out this 👍 
I incorporated this review comment.